src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  color: #3e4e61;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.searchBox {
+  margin: 20px 0;
+}
+
+.searchBar {
+  font-family: inherit;
+  background-image: url(images/Search.svg);
```

Comment:
You should add font-weight: 900 for inputs and font-weight: 300 for placeholder (using ::placeholder)